Downgrade warnings during restore

```
JsonPatch\src\Microsoft.AspNetCore.JsonPatch\project.json(36,53): warning NU1012: Dependency conflict. Microsoft.CSharp 4.3.0-preview1-24530-03 expected System.Reflection.TypeExtensions >= 4.3.0-preview1-24530-03 but received 4.1.0
JsonPatch\src\Microsoft.AspNetCore.JsonPatch\project.json(36,53): warning NU1012: Dependency conflict. System.Dynamic.Runtime 4.3.0-preview1-24530-03 expected System.Reflection.TypeExtensions >= 4.3.0-preview1-24530-03 but received 4.1.0
```

--- a/src/Microsoft.AspNetCore.JsonPatch/project.json
+++ b/src/Microsoft.AspNetCore.JsonPatch/project.json
@@ -33,7 +33,7 @@
     "netstandard1.3": {
       "dependencies": {
         "Microsoft.CSharp": "4.3.0-*",
-        "System.Reflection.TypeExtensions": "4.1.0-*"
+        "System.Reflection.TypeExtensions": "4.3.0-*"
       }
     }
   }

